ember/app/ui/cases/index/controller.js

```diff
@@ -1,13 +1,82 @@
 import Controller from "@ember/controller";
 import { action } from "@ember/object";
+import { debounce } from "@ember/runloop";
+import { inject as service } from "@ember/service";
 import { tracked } from "@glimmer/tracking";
+import calumaQuery from "ember-caluma/caluma-query";
```

Comment:
do you also have to update these import like in the other pr?

Reply:
yes, updated it